phpunit/functional/Glpi/Form/Destination/FormDestinationChangeTest.php

```diff
@@ -35,11 +35,11 @@
 namespace tests\units\Glpi\Form\Destination;
 
 use Glpi\Form\Destination\FormDestinationChange;
-use Glpi\Tests\Form\Destination\AbstractFormDestinationType;
```

Comment:
This class was just renamed (see end of the diff) as it is specific to implementations of `AbstractCommonITILFormDestination`.